white_balancer/src/white_balancer.cpp

```diff
@@ -7,12 +7,13 @@
 #include <dynamic_reconfigure/server.h>
 #include <white_balancer/WhiteBalancerConfig.h>
 #include <boost/bind.hpp>
-
+#include <nodelet/nodelet.h>
+#include <pluginlib/class_list_macros.h>
 
 class WhiteBalancer
 {
 public:
-    WhiteBalancer();
+    WhiteBalancer(ros::NodeHandle nh, ros::NodeHandle pnh);
```

Comment:
Please note why two nodehandlers are needed and that `pnh` stands for private...

Reply:
Done